### ISSUE TITLE
list assets

### ISSUE THREAD

**Issue by deostroll** [yeoman/generator-angular#960]
Need some way to list assets - controllers, services, directives, etc - of an angular project created with yo

**Comment by deostroll** [comment]
You'd have to keep looking at that file always for this...scrolling...its pain...

**Comment by peterblazejewicz** [comment]
@deostroll 
> Need some way to list assets - controllers, services, directives, etc - of an angular project created with yo

Why not use git:
```
git init
```
and then add labels to your commits. See how AngularJS uses meta labels in commit messages:
https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit

**Comment by eddiemonge** [comment]
Im still not sure what is being asked here. Sorry but your description didn't really clarify the problem for me.

**Comment by deostroll** [comment]
There must be an easy way to list what all I've done - for e.g. I've created a controller, I've created some views, directives, etc. Instead of having to go through the file system and finding it all there - there should be some generator way to find them out (cli). Specifically list them on these lines:
- List controllers linked with views/routes
- List all controllers

I don't know but I'd find these things useful while doing angular development
